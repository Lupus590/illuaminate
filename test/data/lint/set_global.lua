-- config: (lint (only (var:set-global)))
local z
x, y, z = 0, 0
