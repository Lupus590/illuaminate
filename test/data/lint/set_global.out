set_global.lua:[3:0-3:1]: Setting unknown global variable "x". [var:set-global]
   │
 3 │ x, y, z = 0, 0
   │ ^
set_global.lua:[3:3-3:4]: Setting unknown global variable "y". [var:set-global]
   │
 3 │ x, y, z = 0, 0
   │    ^
Compilation succeeded with 2 warnings
